src/coreclr/scripts/jitrollingbuild.py

```diff
@@ -262,6 +272,42 @@ def upload_blob(file, blob_name):
 
     files.append(jit_path)
 
+    # Next, look for any and all cross-compilation JITs. These are named, e.g.:
+    #   clrjit_unix_x64_x64.dll
+    #   clrjit_win_arm_x64.dll
+    #   clrjit_win_arm64_x64.dll
+    # and so on, and live in the same product directory as the primary JIT.
+    #
+    # Note that the expression below explicitly filters out the primary JIT since we added that above.
+    # We handle the primary JIT specially so we can error if it is missing. For the cross-compilation
+    # JITs, we don't bother trying to ensure that all the ones we might expect are actually there.
+    #
+    # We don't do a recursive walk because the JIT is also copied to the "sharedFramework" subdirectory,
+    # so we don't want to pick that up.
+
+    if coreclr_args.host_os == "OSX":
+        allowed_extensions = [ ".dylib" ]
+        # Add .dwarf for debug info
+    elif coreclr_args.host_os == "Linux":
+        allowed_extensions = [ ".so" ]
+        # Add .dbg for debug info
+    elif coreclr_args.host_os == "Windows_NT":
+        allowed_extensions = [ ".dll" ]
```

Comment:
curious, why is `allowed_extensions` an array?

Reply:
For future flexibility, such as adding debug info. Currently it doesn't need to be.